packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/FlexboxContainerWidget.tsx

```diff
@@ -248,33 +246,13 @@ export const FlexboxContainerWidget = ({
         onDrop={handleDrop}>
         <Typography variant="body1">{'Drag and drop a widget here'}</Typography>
       </div>
-      <Snackbar
-        anchorOrigin={{
-          vertical: 'bottom',
-          horizontal: 'right',
-        }}
-        open={!!message}
-        autoHideDuration={3000}
+      <Toast
+        message={message}
```

Comment:
Missing `open=` condition if you follow @sbegaudeau's comment ["not to try to be smarter than those who will use us"](https://github.com/eclipse-sirius/sirius-components/pull/1978/files#r1203562508).
There are several other similar instances below, I'm not commenting on each one.